app/src/main/java/nl/tudelft/bbw/controller/CryptoController.java

```diff
@@ -58,4 +64,37 @@ public final String decryptString(String data) {
             throw new RuntimeException("Error - Could not decode string: " + e);
         }
     }
+
+    /**
+     * encryptFile method
+     *
+     * @param path given filepath to encrypt
+     */
+    @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
You are deleting a temporary file, which is needed for the encryption method, since you cannot overwrite the same file.